app/behaviors/navigation-behavior.js

```diff
@@ -48,7 +49,7 @@ define([
      */
     onLinkClicked: function (event) {
       var baseUrl = Settings.get('site_url'),
-        link = event.currentTarget.href;
+          link    = $(event.currentTarget).attr('href');
```

Comment:
The `$` is a bit misleading, `$link` is a string with nothing to do with jQuery.